patches/jsii-rosetta+1.28.0.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/jsii-rosetta/lib/commands/extract.js b/node_modules/jsii-rosetta/lib/commands/extract.js
+index e695ea9..539038e 100644
+--- a/node_modules/jsii-rosetta/lib/commands/extract.js
++++ b/node_modules/jsii-rosetta/lib/commands/extract.js
+@@ -104,7 +104,8 @@ exports.singleThreadedTranslateAll = singleThreadedTranslateAll;
+ async function workerBasedTranslateAll(worker, snippets, includeCompilerDiagnostics) {
+     // Use about half the advertised cores because hyperthreading doesn't seem to help that
+     // much (on my machine, using more than half the cores actually makes it slower).
+-    const N = Math.max(1, Math.ceil(os.cpus().length / 2));
++    // Cap to a reasonable top-level limit to prevent thrash on machines with many, many cores.
++    const N = Math.min(16, Math.max(1, Math.ceil(os.cpus().length / 2)));
```

Comment:
That looks reasonable to me. I'd actually get this one step further when replicating in `jsii-rosetta` for real: have the `16` actually be overridable with something like `JSII_ROSETTA_MAX_WORKER_COUNT`.

Reply:
Yup, makes sense. Done! https://github.com/aws/jsii/pull/2816